### PR TITLE
fix: Fixed switching the organization the branding information is refetched

### LINKED ISSUE #4243: [NV-2756] When switching the organization the branding information is not refetched
## Reproduction Steps

1. Navigate to the Brand page, make sure you have logo or font changed from defaults and saved.
2. Change the organization.
3. The branding settings are not updated.

https://github.com/novuhq/novu/assets/2607232/8a033638-c059-409d-b4fe-1151e143226c

## Definition of done
- The branding reflects currently selected organisation.
- Display branding related to the selected organisation.

### PR DESCRIPTION
**Pull request by vishalkhoje**
### What change does this PR introduce?
This PR fixes the switching of the organization the branding information is refetched.

### Why was this change needed?
This pull request involves incorporating changes into the useEffect function to handle the setting of organization branding values. It addresses a situation where, when switching between organizations, if the initial data such as logo, font family, and color is not defined, the branding field is absent in the data response. This PR adds an else condition to handle this scenario by setting default values for these fields when the branding field is not present.

Fixes #4243

### Other information (Screenshots)

https://github.com/novuhq/novu/assets/36984676/35374364-bff9-4fa1-a7cd-8d2d32859c60

### CODE REVIEW

**Comment by LetItRock** [comment]
hi @vishalkhoje! 👋 thanks for the contribution 🤝
We do value your time and effort, but we would like to give everyone a fair chance to contribute to our repository.
If the person who was assigned to the original ticket submits the PR we will prefer that one, instead of yours. But if the person won't be able to submit the PR or will be unresponsive we will revisit your PR.
For now, I'm closing it.

File: apps/web/src/pages/brand/tabs/BrandingForm.tsx [hunk 1]
```diff
@@ -47,15 +47,11 @@ export function BrandingForm() {
...
-        setValue('image', organization.branding.logo);
-      }
-      if (organization.branding?.color) {
-        setValue('color', organization?.branding?.color);
-      }
-      if (organization.branding?.fontFamily) {
-        setValue('fontFamily', organization?.branding?.fontFamily);
-      }
+      organization?.branding?.logo ? setValue('image', organization.branding.logo) : setValue('image', '');
+      organization?.branding?.color ? setValue('color', organization?.branding?.color) : setValue('color', '#f47373');
```
Comment (davidsoderberg) [inline apps/web/src/pages/brand/tabs/BrandingForm.tsx:51]: Just curious what the default color is here as I do not see it in the old code?

**Review by davidsoderberg — APPROVED** [review]
Looks good. Thanks for contributing 🥳 
Will do a small test tomorrow before merging it.

**Comment by davidsoderberg** [comment]
It works as expected, will merge this now. Thanks for contributing.

**Pull request merged**